pkg/apiserver/rest/utils/cache_test.go

```diff
@@ -25,7 +26,16 @@ import (
 
 var _ = Describe("Test cache utils", func() {
 	It("should return false for IsExpired()", func() {
-		c := NewMemoryCache("test", 10*time.Hour)
+		c := newMemoryCache("test", 10*time.Hour)
 		Expect(c.IsExpired()).Should(BeFalse())
 	})
+
+	It("test cache store", func() {
+		store := NewMemoryCacheStore(context.TODO())
+		store.Put("test", "test data", time.Second*2)
+		store.Put("test2", "test data", 0)
+		time.Sleep(3 * time.Second)
+		Expect(store.Get("test")).Should(BeNil())
+		Expect(store.Get("test2")).Should(Equal("test data"))
+	})
```

Comment:
you need to write more tests, it may have concurrent race bugs.

Reply:
I will add some test cases and comments.